src/prelude/prim.mo

```diff
@@ -376,6 +377,84 @@ func stableMemoryStoreBlob(offset : Nat64, val :  Blob) : () =
 func stableVarQuery() : shared query () -> async {size : Nat64} =
   (prim "stableVarQuery" : () -> (shared query () -> async {size : Nat64})) () ;
 
+// stable regions
+
+// Returns the region assigned to the original stable memory interface.
+func stableMemoryRegion() : Region =
+  (prim "stableMemoryRegion" : () -> Region) ();
+
+func regionNew() : Region =
+  (prim "regionNew" : () -> Region) ();
+
+func regionId(r : Region) : Nat32 = // Only for internal use (compiler tests.)
```

Comment:
Actually, we should not expose this at all (or have a separate, increasing logical id) if we ever do GC.
So remove for now (and maybe leave space for a 32-bit logical id an allocation counter) in metadata
We could expose equals though ( but not comparison).

Reply:
Done